src/azure-cli-core/azure/cli/core/__init__.py

```diff
@@ -489,6 +489,25 @@ def _get_extension_suppressions(mod_loaders):
 
         return self.command_table
 
+    def sort_command_loaders(self, command_loaders):
```

Comment:
Making it a protected static method sounds more proper:
```suggestion
    @staticmethod
    def _sort_command_loaders(command_loaders):
```